docs/clustering/multi-zone-setup.rst

```diff
@@ -5,124 +5,188 @@ CrateDB multi-zone setup
 ========================
 
 If possible, we recommend running all CrateDB nodes of a cluster inside the
-same physical space (e.g. data center) to minimize network latency and maximize
+same physical space (i.e. data center) to minimize network latency and maximize
 speed between the nodes. These factors can have a significant impact on the
 performance of CrateDB clusters.
 
-This is because replicas are written *synchronously* and making a write task
-wait for the replicas to write somewhere in a data center hundreds of miles
-away can lead to noticeable latency and cause the cluster to slow down.
+This is because replicas are written *synchronously* and making a `write 
+operation`_ wait for all the replicas to write somewhere in a data center
+hundreds of miles away can lead to noticeable latency and cause the cluster to
+slow down.
 
-In certain scenarios it may be necessary to run a cluster across multiple data
-centers or availability zones. We'll call these **zones** from now on.
+In some cases, it may be necessary to run a cluster across multiple data
+centers or availability zones (*zones*, for short). This guide shows you
+how to set up a multi-zone CrateDB cluster.
 
 .. rubric:: Table of contents
 
 .. contents::
    :local:
 
-Requirements
-============
+.. _multi-zone-requirements:
 
-For a multi zone setup, CrateDB clusters need to fulfill the following:
+Multi-zone requirements
+=======================
 
-1. When inserting data, it should be replicated in a way so that at least one
-   full replica is present inside each zone.
-2. If a zone becomes unavailable, all data still needs to be fully available.
+For a multi-zone setup, CrateDB clusters need to fulfill the following:
+
+1. Data inserts should be replicated in a way where at least one full `replica`_
+   is present in each zone.
+2. All data still needs to be fully available if a zone becomes unavailable.
```

Comment:
please add an empty line between list items that are full sentences

(here and above)